### PR TITLE
made more explicit that RAML does not allow multiple paths to be resolved into the same

### PR DESCRIPTION
**Pull request by sichvoge**
According to #367

### CODE REVIEW

File: versions/raml-10/raml-10.md [hunk 1]
```diff
@@ -1671,6 +1671,26 @@ https://api.github.com/users/{userId}/keys
 https://api.github.com/users/{userId}/keys/{keyId}
 ```
 
+A RAML processor MUST not allow a fully-combined path, which you get from combining the resource's path with all of its parents, to be identical to another fully-combined path, WITHOUT consideration to the possible values of any URI parameter.
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: MUST not --> MUST NOT

Comment (usarid) [inline versions/raml-10/raml-10.md]: We should find a better way to say it. 

First, define a path expansion algorithm. I think we already have that, in the definition of deriving the absolute URI for each resource -- but we should tweak that section so it does not imply you have to have a `baseUri` to proceed, and gives you a path expansion algorithm you can apply with or without a `baseUri`.

Once you're done that, refer to that algorithm here, and say that two fully-resolved paths cannot be identical, when they're compared with each other without resolving (substituting the values of) any URI parameters.

**Comment by usarid** [comment]
See comments.

**Comment by sichvoge** [comment]
See new commit.

**Comment by usarid** [comment]
Needs more changes per our Slack

File: versions/raml-10/raml-10.md [hunk 2]
```diff
@@ -1622,7 +1622,7 @@ baseUri: https://api.github.com
 
 The key of a resource property, i.e. its relative URI, MAY consist of multiple URI path fragments separated by slashes; e.g. /bom/items may indicate the collection of items in a bill of materials as a single resource. However, if the individual URI path fragments are themselves resources, the API definition SHOULD use nested resources to describe this structure; e.g. if /bom is itself a resource then /items should be a nested resource of /bom, vs using /bom/items as a non-nested resource.
 
-Absolute URIs are not explicitly specified. They are computed by starting with the baseUri and appending the relative URI of the top-level resource, and then successively appending the relative URI values for each nested resource until the target resource is reached.
+Absolute URIs are not explicitly specified. They are computed by appending the relative URI of the top-level resource, and then successively appending the relative URI values for each nested resource until the target resource is reached. If and only if the baseUri is defined, the baseUri is appended to the first segment, which represents the top-level resource, of relative URIs.
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: Change
>If and only if the baseUri is defined, the baseUri is appended to the first segment, which represents the top-level resource, of relative URIs.
to
>In this formation of the absolute URI, if a baseUri is defined, it is prepended before the relative URI of the top-level resource; any trailing slashes in the baseUri are removed before prepending.

File: versions/raml-10/raml-10.md [hunk 3]
```diff
@@ -1671,6 +1671,26 @@ https://api.github.com/users/{userId}/keys
 https://api.github.com/users/{userId}/keys/{keyId}
 ```
 
+A RAML processor MUST NOT allow one of the computed absolute URIs to be identical to another one, without consideration to the possible values of any URI parameter.
```
Comment (usarid) [inline versions/raml-10/raml-10.md]: Change

>A RAML processor MUST NOT allow one of the computed absolute URIs to be identical to another one, without consideration to the possible values of any URI parameter.

to

>A RAML processor MUST NOT allow one of the computed absolute URIs to be identical to another one; comparison of absolute URIs is done without consideration to the possible values of any URI parameter, i.e. any URI parameters are not expanded or evaluated but rather left as is.

**Comment by usarid** [comment]
See my comments, otherwise LGTM.

**Pull request merged**